sklearn/tests/test_grid_search.py

```diff
@@ -217,23 +217,6 @@ def test_grid_search_score_method():
                                               scoring='roc_auc').fit(X, y)
     search_auc = GridSearchCV(clf, grid, scoring='roc_auc').fit(X, y)
 
-    # Check warning only occurs in situation where behavior changed:
-    # estimator requires score method to compete with scoring parameter
-    score_no_scoring = assert_no_warnings(search_no_scoring.score, X, y)
-    score_accuracy = assert_warns(ChangedBehaviorWarning,
-                                  search_accuracy.score, X, y)
-    score_no_score_auc = assert_no_warnings(search_no_score_method_auc.score,
-                                            X, y)
-    score_auc = assert_warns(ChangedBehaviorWarning,
-                             search_auc.score, X, y)
-    # ensure the test is sane
```

Comment:
@jnothman said "I suspect we want this still" And that was referring to the "ensure the test is sane" part, which will pass without error.